src/detection/card/desktop-style.js

```diff
@@ -0,0 +1,10 @@
+import { makeStyles } from '@material-ui/core'
+
+const root = {
```

Comment:
![image](https://user-images.githubusercontent.com/23569938/126044169-a15c3c4b-5497-43e1-a08b-37499cd043df.png)
Other than the `width`, these styles are identical. Can we maybe extract the other ones into a common attribute?

Reply:
Done